CRM/Logging/ReportDetail.php

```diff
@@ -218,25 +221,68 @@ protected function convertDiffsToRows() {
           $to = implode(', ', array_filter($tos));
         }
 
+        $cfArray = [];
         $tableDAOClass = CRM_Core_DAO_AllCoreTables::getClassForTable($table);
+        $fkClassName = NULL;
         if (!empty($tableDAOClass)) {
           $tableDAOFields = (new $tableDAOClass())->fields();
           // If this field is a foreign key, then we can later use the foreign
           // class to translate the id into something more useful for display.
           $fkClassName = $tableDAOFields[$field]['FKClassName'] ?? NULL;
         }
+        else {
+          /* Since this table didn't match a core table, check if it's a custom field. */
```

Comment:
Typically we use `//` for a single-line comment
```suggestion
          // Since this table didn't match a core table, check if it's a custom field.
```